profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -199,13 +219,114 @@ fun UserTabs(modifier: Modifier = Modifier) {
                         )
                     )
                 )
-        ) {
-            Text(
-                text = stringResource(coreR.string.coming_soon),
-                color = MaterialTheme.colorScheme.onSurfaceVariant,
-                style = MaterialTheme.typography.labelLarge,
-                modifier = Modifier.align(Alignment.Center)
+        ) { page ->
+            Box(Modifier.fillMaxSize()) {
+                when (ProfileTabs.entries[page]) {
+                    ProfileTabs.ABOUT -> AboutTab(viewer)
+                    else -> Text(
+                        text = stringResource(coreR.string.coming_soon),
+                        color = MaterialTheme.colorScheme.onSurfaceVariant,
+                        style = MaterialTheme.typography.labelLarge,
+                        modifier = Modifier.align(Alignment.Center)
+                    )
+                }
+            }
+        }
+    }
+}
+
+enum class ProfileTabs(val title: String) {
+    ABOUT("About"),
+    ANIME("Anime"),
+    MANGA("Manga"),
+    FAVOURITES("Fave"),
+    STATISTICS("Stats")
+}
+
+@Composable
+fun AboutTab(
+    viewer: Viewer,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.padding(
+            top = LocalPaddings.current.large,
+            start = LocalPaddings.current.large,
+            end = LocalPaddings.current.large
+        ),
+        verticalArrangement = Arrangement.spacedBy(LocalPaddings.current.medium)
+    ) {
+        Genres(viewer.genres)
+    }
+}
+
+@Composable
+fun Genres(
+    genres: List<Viewer.Genre>,
+    modifier: Modifier = Modifier
+) {
+    val barWidthAnimation = remember { Animatable(0f) }
+    val barAlphaAnimation = remember { Animatable(0f) }
+    val barColor = MaterialTheme.colorScheme.primary
+    LaunchedEffect("barAnimation") {
+        launch {
+            barWidthAnimation.animateTo(
+                targetValue = 1f,
+                animationSpec = tween(durationMillis = 1000, delayMillis = 250)
+            )
+        }
+        launch {
+            barAlphaAnimation.animateTo(
+                targetValue = 1f,
+                animationSpec = tween(durationMillis = 1000, delayMillis = 250)
             )
         }
     }
+
+    Text(
+        text = "Genres",
+        style = MaterialTheme.typography.titleMedium
+    )
+    Row(modifier.height(IntrinsicSize.Max)) {
+        Column(horizontalAlignment = Alignment.End) {
+            genres.forEach {
+                Text(
+                    text = it.genre,
+                    style = m3TextStyles().textStyle.copy(
+                        color = MaterialTheme.colorScheme.onBackground.copy(alpha = 0.74f)
```

Comment:
What the magic alpha value doing 👀

Reply:
I'll fix this in a follow up PR for typography, a couple other files have this problem.